src/routers/datatypeRouters/edges/variants_phenotypes.ts

```diff
@@ -1,16 +1,31 @@
 import { z } from 'zod'
 import { publicProcedure } from '../../../trpc'
 import { loadSchemaConfig } from '../../genericRouters/genericRouters'
-import { variantsQueryFormat } from '../nodes/variants'
 import { studyFormat } from '../nodes/studies'
-import { getDBReturnStatements, getFilterStatements, paramsFormatType, preProcessRegionParam } from '../_helpers'
+import { getDBReturnStatements, getFilterStatements, paramsFormatType } from '../_helpers'
 import { descriptions } from '../descriptions'
 import { QUERY_LIMIT } from '../../../constants'
 import { db } from '../../../database'
 import { TRPCError } from '@trpc/server'
+import { variantIDSearch } from '../nodes/variants'
 
 const MAX_PAGE_SIZE = 100
 
+const variantsQueryFormat = z.object({
+  variant_id: z.string().trim().optional(),
+  spdi: z.string().trim().optional(),
+  hgvs: z.string().trim().optional(),
+  rsid: z.string().trim().optional(),
+  chr: z.string().trim().optional(),
+  position: z.string().trim().optional(),
+  phenotype_id: z.string().trim().optional(),
+  log10pvalue: z.string().trim().optional(),
+  organism: z.enum(['Homo sapiens']),
+  verbose: z.enum(['true', 'false']).default('false'),
+  page: z.number().default(0),
+  limit: z.number().optional()
+})
+
```

Comment:
why didn't the extension of variantsQueryFormat from the original variants.ts file work?

Reply:
I like the idea to reuse the variantsQueryFormat from variants.ts. There are a couple issues for me: first you need to remove some fields and merge more objects to update the query format, which can be error-prone. Second, the order of the fields after remove and merge is not the order I want. The order of the params for each endpoint right now is inconsistent, we may need to figure a rule to decide the order of certain type of params.